Tests/MessagingInApp/MessagingInAppIntegrationTest.swift

```diff
@@ -0,0 +1,20 @@
+@testable import CioMessagingInApp
+@testable import CioTracking
+@testable import Common
+import Foundation
+import SharedTests
+import XCTest
+
+class MessagingInAppIntegrationTests: IntegrationTest {
+    // Bug reported when: using in-app messaging, debug logging enabled, and using iOS version 1.2.8.
```

Comment:
I think it is good to mention that the bug is not present in 1.2.7 but just 1.2.8.